tests/ConcordiumNetSdk.IntegrationTests/ConcordiumNodeClientTests/GetAccountInfoAsyncTests.cs

```diff
@@ -1,5 +1,5 @@
-﻿using System.Threading.Tasks;
-using ConcordiumNetSdk.Responses.AccountInfoResponse;
+﻿/*
```

Comment:
Please provide documentation on what is being tested, what the success means.

Reply:
This is essentially leftovers from the old repository with a few changes?

I'll add tests today, in a different PR